googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2505,6 +2506,7 @@ typedef TypeWithSize<4>::Int Int32;
 typedef TypeWithSize<4>::UInt UInt32;
 typedef TypeWithSize<8>::Int Int64;
 typedef TypeWithSize<8>::UInt UInt64;
+typedef TypeWithSize<sizeof(void*)>::UInt UIntPtr;
```

Comment:
Are you all okay using `uintptr_t`? If so, that works for me. My assumption was that, given the rest of these` typedef`s existed, you wanted to support platforms which were missing some of these.